src/sentry/api/serializers/models/activity.py

```diff
@@ -10,6 +10,9 @@
 
 @register(Activity)
 class ActivitySerializer(Serializer):
+    def __init__(self, environment_id_func=None):
+        self.environment_id_func = environment_id_func if environment_id_func is not None else lambda: None
```

Comment:
This should be `None` and allow the `GroupSerializer` constructor to manage the default value.